src/server/worldserver/worldserver.conf.dist

```diff
@@ -2843,6 +2843,23 @@ AllowTrackBothResources = 0
 
 PlayerStart.AllReputation = 0
 
+#
+#    Weekend.XP.Days
+#        Description: Enables XP increase during last days of the week.
+#
+#        Default:     0 - (Disabled)
+#                     2 - (Weekend - Saturdays and Sundays)
+#                     7 - (Always active)
```

Comment:
Wouldn't it make more sense to just have 0 and 1 as values, for if weekend bonus experience should be applied or not?
7 for "Always active" doesn't make much sense, you'd just use the other config for xp rates.
0, 2 and 7 doesn't make much sense tbh.

There's also another option of making this a bitmask, allowing any week day to be chosen as the bonus xp "weekend".
But then joschiwald's game_event suggestion starts to make more sense.

Either way I'd remove the "Always" active options and stick with 0 and 1 as values for bonus weekend is "not active"/"active"